### ISSUE TITLE
Support more players

### ISSUE THREAD

**Issue by jpeterbaker** [jpeterbaker/bga_homeworlds#16]
This will be quite an undertaking, but it would be nice to let more people play together with various victory conditions.
- Sinister
- Good vs evil
- Teams
- Last captain standing

**Comment by SimonN** [comment]
Multiplayer needs a rule clarification: Are former homeworlds of eliminated players still homeworlds? This matters for fading.

No ruling can be perfect:
* Assume former homeworlds stay homeworlds. Then one-star homeworlds are almost, but not quite, like other basic systems. We'll have to remember exactly which single stars are former homeworlds because they fade at end of turn.
* Assume former homeworlds become basic systems. Then you'll have binary basic systems. You can't leave this system and then rediscover a same-looking system during a turn, even if its pieces are available in the bank.
* Assume former homeworlds remain homeworlds as long as they still have both stars, and become basic stars immediately when one of the stars is destroyed. This might capture initial feelings better, but it's the least consistent ruling.

**Comment by jpeterbaker** [comment]
I agree that my rules page needs a clarification, although strictly speaking my website with the rules is in a different repo.
I don't think there's anything official on this rule, but I lean heavily toward your second option. Once no extant civilization is devoted to a system, it can be forgotten as easily as any other system.

**Comment by jpeterbaker** [comment]
I talked to John about the fate of homeworlds of eliminated players, and he agrees that there's nothing special about the former homeworld of an eliminated player.
